benchmark/Utilities.hpp

```diff
@@ -4,9 +4,9 @@
 #include <time.h>
 #include <stdlib.h>
 
-static std::unordered_map<unsigned long, CXXGRAPH::Node<int> *> generateRandomNodes(unsigned long numberOfNodes, int MaxValue)
+static std::map<unsigned long, CXXGRAPH::Node<int> *> generateRandomNodes(unsigned long numberOfNodes, int MaxValue)
```

Comment:
Is there a good reason to change from unordered_map to map?? Map is a bit less efficient when operating through elements.
I know there are issues when working with unordered_map keys when trying to use some algorithms over them for hashes.
I guess there were problems using unordered to motivate this change.

Reply:
I change this, because the find over unordered map return bad results